.prettierignore

```diff
@@ -3,3 +3,4 @@
 .parcel-cache
 .github
 **/protos
+**/coverage
```

Comment:
non-blocking: Possible workaround to avoid duplication I've seen was multiple times was `prettier ----ignore-path .eslingignore` in `package.json`.

Reply:
What about code editors? Would they respect that?